geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreFactory.scala

```diff
@@ -114,6 +117,7 @@ class HBaseDataStoreFactory extends DataStoreFactorySpi with LazyLogging {
 object HBaseDataStoreParams {
   val BigTableNameParam    = new Param("bigtable.table.name", classOf[String], "Table name", true)
   val ConnectionParam      = new Param("connection", classOf[Connection], "Connection", false)
+  val CoprocessorUrl       = new Param("coprocessor.url", classOf[Path], "Coprocessor Url", false, new Path("hdfs://"))
```

Comment:
How do you mean? It gives you a string input field if that's what you're asking.